### ISSUE TITLE
Qwen long context query produces garbage response

### ISSUE THREAD

**Issue by micoscriptor** [ollama/ollama#2580]
I really appreciate the Ollama project - many thanks to everyone involved in this awesome software!

Unfortunately I believe I have just found an issue. The recently published Qwen 1.5 model supports a 32k context length which is really nice. This supposedly should work for all variants from 0.5B all the way up to 72B. I tested the "Qwen:1.8b-chat" model  both on LM Studio and in Ollama using a long query, almost 7k tokens, and it works just fine in LM Studio but produces strange garbage in Ollama. I can't understand why? I have adjusted the context length to 12000 tokens in both applications. For Ollama I created a new model using this ModelFile:

```
FROM qwen:1.8b-chat

PARAMETER num_ctx 12000
```

I get a very good response from LM studio but Ollama responds with this text: "German and both, the, the, , its in the ( perhaps, however, in, it, German as, the command, the, the or, both and compare, 
both, the, Germany as the , Japan Germany , a to to to, to, to, the, the, the, , the, the, the, , as to, to, the, as, The, the 
the command, as ( the: the, the: German: the strategic, the, military, or, its and, ( both, command, it, as, , the in, 
Germany,,,, the, ,, the,, the, the,,,: the, to: the, the, the the, in ( the, the, " the, the war, " and, and, German, the, the, 
both, the the, its, and,, and, as, the, the, the, the, the, command, the German, and, the, and, its,, ,, and, the, and, the, 
Japan,, (, the, German,, and, Germany, its, it, German, with, the respective, their, the, c, German, German, both, to, the, the,
and, the, the the, as, the, to the, the, the, to, the the, the the German, the both轻轻地 the, , it, Germany, the, its command, 
but, the, the all German , the respective war, as the weapons,, aircraft, their, the, the German, and, the, however, it, it, 
the, in, both, to the, command, and, the, the, the, the, the the, that, the, the, as, the, German, the, the , the, the, the, 
the, the,, the respective, the, the, the, as the the aircraft, the the , the German all, the, and the command, the, and"

The full query below:
(it is from this Wikipedia page: https://en.wikipedia.org/wiki/Air_warfare_of_World_War_II)

```
"""Based on the context below, what was the main disadvantage of the Luftwaffe?

============================================

Air warfare was a major component in all theaters of World War II and, together with anti-aircraft warfare, consumed a large fraction of the industrial output of the major powers. Germany and Japan depended on air forces that were closely integrated with land and naval forces; the Axis powers downplayed the advantage of fleets of strategic bombers and were late in appreciating the need to defend against Allied strategic bombing. By contrast, Britain and the United States took an approach that greatly emphasized strategic bombing and (to a lesser degree) tactical control of the battlefield by air as well as adequate air defenses. Both Britain and the U.S. built substantially larger strategic forces of large, long-range bombers. Simultaneously, they built tactical air forces that could win air superiority over the battlefields, thereby giving vital assistance to ground troops. The U.S. Navy and Royal Navy also built a powerful naval-air component based on aircraft carriers, as did the Imperial Japanese Navy; these played the central role in the war at sea.[1]

Pre-war planning
Before 1939, all sides operated under largely theoretical models of air warfare. Italian theorist Giulio Douhet in the 1920s summarised the faith that airmen during and after World War I developed in the efficacy of strategic bombing. Many said it alone could win wars,[2] as "the bomber will always get through". The Americans were confident that the Boeing B-17 Flying Fortress bomber could reach targets, protected by its own weapons, and bomb, using the Norden bombsight, with "pickle barrel" accuracy.[3] Japanese aviation pioneers felt that they had developed the finest naval aviators in the world.

Air forces

Germany: Luftwaffe
Main article: Luftwaffe

A Messerschmitt Bf 109 escorting a Junkers Ju 87 of the Luftwaffe in 1941
The Luftwaffe was the aerial warfare branch of the Wehrmacht. Under the leadership of Hermann Göring, it was able to learn and test new combat techniques in the Spanish Civil War. The war also led to greater emphasis on anti-air weapons and fighter aircraft due to their ability to defend against enemy bombers.[4] Its advanced technology and rapid growth led to exaggerated fears in the 1930s that helped to persuade the British and French into appeasement. In the war the Luftwaffe performed well in 1939–41, as its Stuka dive bombers terrified enemy infantry units. But the Luftwaffe was poorly coordinated with overall German strategy, and never ramped up to the size and scope needed in a total war, partly due to a lack of military aircraft production infrastructure for both completed airframes and powerplants when compared to either the Soviet Union or the United States. The Luftwaffe was deficient in radar technology except for their usable UHF and later VHF band airborne intercept radar designs such as the Lichtenstein and Neptun radar systems for their night fighters. The Messerschmitt Me 262 jet fighter did not enter service until July 1944, and the lightweight Heinkel He 162 appeared only during the last months of the air war in Europe. The Luftwaffe could not deal with Britain's increasingly lethal defensive fighter screen after the Battle of Britain, or the faster P-51 Mustang escort fighters after 1943.

When the Luftwaffe's fuel supply ran dry in 1944 due to the oil campaign of World War II, it was reduced to anti-aircraft flak roles, and many of its men were sent to infantry units. By 1944 it operated 39,000 flak batteries staffed with a million people in uniform, both men and women.

The Luftwaffe lacked the bomber forces for strategic bombing, because it did not think such bombing was worthwhile, especially following the June 3, 1936, death of General Walther Wever, the prime proponent of a strategic bomber force for the Luftwaffe. They did attempt some strategic bombing in the east with the problematic Heinkel He 177A. Their one success was destroying an airbase at Poltava Air Base, Ukraine during the Allied Operation Frantic, which housed 43 new B-17 bombers and a million tons of aviation fuel.[5]

Introduction of turbojet-powered combat aircraft, mostly with the Messerschmitt Me 262 twin-jet fighter, the Heinkel He 162 light jet fighter and the Arado Ar 234 reconnaissance-bomber was pioneered by the Luftwaffe, but the delayed period (1944–45) of their introduction – much of which was due to the lengthy development time for both the BMW 003 and Junkers Jumo 004 jet engine designs—as well as the failure to produce usable examples of their two long-developed higher-power aviation engines, the Junkers Jumo 222 multibank 24-cylinder piston engine of some 2,500 hp, and the advanced Heinkel HeS 011 turbojet of nearly 2,800 lb. of thrust, each of which were meant to power many advanced German airframe design proposals in the last years of the war—meant that they were introduced "too little, too late", as so many other advanced German aircraft designs (and indeed, many other German military weapon systems) had been during the later war years.

Although Germany's allies, especially Italy and Finland, had air forces of their own, there was very little coordination with them. Not until very late in the war did Germany share its aircraft and alternative fuel blueprints and technology with its ally Japan, resulting in the Nakajima Kikka jet fighter and the Mitsubishi Shusui rocket fighter, respectively based on the Me 262A and Me 163B—both of which, similarly, came far too late for Japan to improve its defensive aircraft systems, or to make alternative fuels and lubricants.[6]

Britain: Royal Air Force
Main article: History of the Royal Air Force

Hawker Hurricanes and Supermarine Spitfires of the RAF flying in formation in 1940
The British had their own very well-developed theory of strategic bombing, and built the long-range bombers to implement it.[7]

Once it became clear that Germany was a threat, the RAF started on a large expansion, with many airfields being set up and the number of squadrons increased. From 42 squadrons with 800 aircraft in 1934, the RAF had reached 157 squadrons and 3,700 aircraft by 1939.[8] They combined the newly developed radar with communications centres to direct their fighter defences. Their medium bombers were capable of reaching the German industrial centre of the Ruhr, and larger bombers were under development.


Aviation goggles and flying helmet used by famous RAF pilot Billy Strachan (1921-1998)
The RAF underwent rapid expansion following the outbreak of war against Germany in 1939. This included the training in other Commonwealth nations (particularly Canada) of half of British and Commonwealth aircrews, some 167,000 men in all. It was the second largest in Europe. The RAF also integrated Polish and other airmen who had escaped from Hitler's Europe. In Europe, the RAF was in operational control of Commonwealth aircrews and Commonwealth squadrons although these retained some degree of independence (such as the formation of No. 6 Group RCAF to put Canadian squadrons together in a nationally identifiable unit).

The RAF had three major combat commands based in the United Kingdom: RAF Fighter Command charged with defence of the UK, RAF Bomber Command (formed 1936) which operated the bombers that would be offensive against the enemy, and RAF Coastal Command which was to protect Allied shipping and attack enemy shipping. The Royal Navy's Fleet Air Arm operated land-based fighters in defence of naval establishments and carrier-based aircraft. Later in the war the RAF's fighter force was divided into two Air Defence of Great Britain (ADGB) for protecting the UK and the Second Tactical Air Force for ground offensive support in the North West Europe campaign.

Bomber Command participated in two areas of attack – the strategic bombing campaign against German war production, and the less well known mining of coastal waters off Germany (known as Gardening) to contain its naval operations and prevent the U-boats from freely operating against Allied shipping. In order to attack German industry by night the RAF developed navigational aids, tactics to overwhelm the German defences control system, tactics directly against German night-fighter forces, target marking techniques, many electronic aids in defence and attack, and supporting electronic warfare aircraft. The production of heavy aircraft competed with resources for the Army and the Navy, and it was a source of disagreement as to whether the effort could be more profitably expended elsewhere.

Increasingly heavy losses during the latter part of 1943 due to the reorganized Luftwaffe night fighter system (Wilde Sau tactics), and Sir Arthur Harris' costly attempts to destroy Berlin in the winter of 1943/44, led to serious doubts as to whether Bomber Command was being used to its fullest potential. In early 1944 the UK air arm was put under Eisenhower's direct control where it played a vital role in preparing the way for the Overlord Invasion.[9][10]

Soviet Union: Soviet Air Force
Main articles: Soviet Air Forces § World War II, and Eastern Front (World War II)

Ilyushin Il-2s of the Soviet Air Forces near Moscow in 1943
By the end of the war, Soviet annual aircraft production had risen sharply with annual Soviet production peaking at 40,000 aircraft in 1944. Some 157,000 aircraft were produced, of which 126,000 were combat types for the Voyenno-Vozdushnye Sily or VVS (as the Soviet Union named their air arm), while the others were transports and trainers.[11][12] The critical importance of the ground attack role in defending the Soviet Union from the Axis' Operation Barbarossa through to the final defeat of Nazi Germany with the Battle of Berlin resulted in the Soviet military aviation industry creating more examples of the Ilyushin Il-2 Shturmovik during the war than any other military aircraft design in aviation history, with just over 36,000 examples produced.[13]

During the war the Soviets employed 7500 bombers to drop 30 million bombs on German targets, with a density that sometimes reached 100–150 tons/ sq kilometer.[14][15]

United States: Army Air Forces
Main article: United States Army Air Forces

A Consolidated Liberator I flying over Curtiss P-40 Warhawks of the U.S. Army Air Forces in 1943
Before the attack on Pearl Harbor and during the period within which the predecessor U.S. Army Air Corps became the Army Air Forces in late June 1941, President Franklin D. Roosevelt gave command of the Navy to an aviator, Admiral Ernest King, with a mandate for an aviation-oriented war in the Pacific. FDR allowed King to build up land-based naval and Marine aviation, and seize control of the long-range bombers used in antisubmarine patrols in the Atlantic. Roosevelt basically agreed with Robert A. Lovett, the civilian Assistant Secretary of War for Air, who argued, "While I don't go so far as to claim that air power alone will win the war, I do claim the war will not be won without it."[16]

Army Chief of Staff George C. Marshall rejected calls for complete independence for the Air Corps, because the land forces generals and the Navy were vehemently opposed. In the compromise that was reached it was understood that after the war, the aviators would get their independence. Meanwhile, the Air Corps became the Army Air Forces (AAF) in June, 1941, combining all their personnel and units under a single commanding general, an airman. In 1942 the Army reorganized into three equal components, one of which was the AAF, which then had almost complete freedom in terms of internal administration. Thus the AAF set up its own medical service independent of the Surgeon General, its own WAC units, and its own logistics system. It had full control over the design and procurement of airplanes and related electronic gear and ordnance. Its purchasing agents controlled 15% of the nation's Gross National Product. Together with naval aviation, it recruited the best young men in the nation. General Henry H. Arnold headed the AAF. One of the first military men to fly, and the youngest colonel in World War I, he selected for the most important combat commands men who were ten years younger than their Army counterparts, including Ira Eaker (b. 1896), Jimmy Doolittle (b. 1896), Hoyt Vandenberg (b. 1899), Elwood "Pete" Queseda (b. 1904), and, youngest of them all, Curtis LeMay (b. 1906). Although a West Pointer himself, Arnold did not automatically turn to Academy men for top positions. Since he operated independent of theatre commanders, Arnold could and did move his generals around, and speedily removed underachievers.[17]

Aware of the need for engineering expertise, Arnold went outside the military and formed close liaisons with top engineers like rocket specialist Theodore von Karmen at Caltech. Arnold was given seats on the US Joint Chiefs of Staff and the US-British Combined Chiefs of Staff. Arnold, however, was officially Deputy Chief of [Army] Staff, so on committees he deferred to his boss, General Marshall. Thus Marshall made all the basic strategic decisions, which were worked out by his "War Plans Division" (WPD, later renamed the Operations Division). WPD's section leaders were infantrymen or engineers, with a handful of aviators in token positions.[18]

The AAF had a newly created planning division, whose advice was largely ignored by WPD. Airmen were also underrepresented in the planning divisions of the Joint Chiefs of Staff and of the Combined Chiefs. Aviators were largely shut out of the decision-making and planning process because they lacked seniority in a highly rank-conscious system. The freeze intensified demands for independence, and fueled a spirit of "proving" the superiority of air power doctrine. Because of the young, pragmatic leadership at the top, and the universal glamor accorded aviators, morale in the AAF was strikingly higher than anywhere else (except perhaps Navy aviation).

The AAF provided extensive technical training, promoted officers and enlisted faster, provided comfortable barracks and good food, and was safe, with an American government-sponsored pilot training program in place as far back as 1938, that did work in concert when necessary with the British Commonwealth's similar program within North America. The only dangerous jobs were voluntary ones as crew of fighters and bombers—or involuntary ones at jungle bases in the Southwest Pacific. Marshall, an infantryman uninterested in aviation before 1939, became a partial convert to air power and allowed the aviators more autonomy. He authorized vast spending on planes, and insisted that American forces had to have air supremacy before taking the offensive. However, he repeatedly overruled Arnold by agreeing with Roosevelt's requests in 1941–42 to send half of the new light bombers and fighters to the British and Soviets, thereby delaying the buildup of American air power.[19]

The Army's major theatre commands were given to infantrymen Douglas MacArthur and Dwight D. Eisenhower. Neither had paid much attention to aviation before the war. However the air power advocate Jimmy Doolittle succeeded Eaker as 8th Air Force commander at the start of 1944. Doolittle instituted a critical change in strategic fighter tactics, and the 8th Air Force bomber raids faced less and less Luftwaffe defensive fighter opposition for the rest of the war.

Offensive counter-air, to clear the way for strategic bombers and an eventually decisive cross-channel invasion, was a strategic mission led by escort fighters partnered with heavy bombers. The tactical mission, however, was the province of fighter-bombers, assisted by light and medium bombers.

American theatre commanders became air power enthusiasts and built their strategies around the need for tactical air supremacy. MacArthur had been badly defeated in the Philippines in 1941–42 primarily because the Japanese controlled the sky. His planes were outnumbered and outclassed, his airfields shot up, his radar destroyed, and his supply lines cut. His infantry never had a chance. MacArthur vowed never again. His island-hopping campaign was based on the strategy of isolating Japanese strongholds while leaping past them. Each leap was determined by the range of his 5th Air Force, and the first task on securing an objective was to build an airfield to prepare for the next leap.[20][21] Eisenhower's deputy at SHAEF was Air Chief Marshal Arthur Tedder, who had been commander of the Allied Mediterranean Air Command when Eisenhower was in charge of Allied operations in the Mediterranean.

Doctrine and technology
The Allies won battlefield air supremacy in the Pacific in 1943, and in Europe in 1944. That meant that Allied supplies and reinforcements would get through to the battlefront, but not the enemy's. It meant the Allies could concentrate their strike forces wherever they pleased, and overwhelm the enemy with a preponderance of firepower. There was a specific campaign, within the overall strategic offensive, for suppression of enemy air defences, or, specifically, Luftwaffe fighters.

Aircrew training

CAC Wackett trainers of the Royal Australian Air Force in 1942
While the Japanese began the war with a superb set of naval aviators, trained at the Misty Lagoon experimental air station, their practice, perhaps from the warrior tradition, was to keep the pilots in action until they died. The U.S. position, at least for naval aviation, was a strict rotation between sea deployments and shore duty, the latter including training replacements, personal training, and participating in doctrinal development. The U.S. strategic bombing campaign against Europe did this in principle, but relatively few crews survived the 25 missions of a rotation. On December 27, 1938, the United States had initiated the Civilian Pilot Training Program to vastly increase the number of ostensibly "civilian" American pilots, but this program also had the eventual effect of providing a large flight-ready force of trained pilots for future military action if the need arose.

Other countries had other variants. In some countries, it seemed to be a matter of personal choice if one stayed in combat or helped build the next generation. Even where there was a policy of using skills outside combat, some individuals, e.g. Guy Gibson VC insisted on returning to combat after a year. Both Gibson's successors at 617 Squadron were ordered off "ops" permanently – Leonard Cheshire VC after 102 operations, "Willie" Tait (DSO & 3 Bars) after 101 – reflecting the strain of prolonged operations.

The British Commonwealth Air Training Plan (and related schemes) as well as training British crew in North America, away from the war, contributed large numbers of aircrew from outside the UK to the forces under RAF operational control. The resulting "Article XV squadrons" nominally part of individual Commonwealth air forces were filled from a pool of mixed nationalities. While RAF Bomber Command let individuals form teams naturally and bomber aircrew were generally heterogeneous in origins, the Canadian government pushed for its bomber aircrew to be organised in one Group for greater recognition – No. 6 Group RCAF.

Logistics
Airfield construction

RAF construction workers finishing the construction of an airfield near Lingèvres in 1944
Arnold correctly anticipated that the U.S. would have to build forward airfields in inhospitable places. Working closely with the Army Corps of Engineers, he created Aviation Engineer Battalions that by 1945 included 118,000 men. Runways, hangars, radar stations, power generators, barracks, gasoline storage tanks, and ordnance dumps had to be built hurriedly on tiny coral islands, mud flats, featureless deserts, dense jungles, or exposed locations still under enemy artillery fire. The heavy construction gear had to be imported, along with the engineers, blueprints, steel-mesh landing mats, prefabricated hangars, aviation fuel, bombs and ammunition, and all necessary supplies. As soon as one project was finished the battalion would load up its gear and move forward to the next challenge, while headquarters inked in a new airfield on the maps.[22]

The engineers opened an entirely new airfield in North Africa every other day for seven straight months. Once when heavy rains along the coast reduced the capacity of old airfields, two companies of Airborne Engineers loaded miniaturized gear into 56 transports, flew a thousand miles to a dry Sahara location, started blasting away, and were ready for the first B-17 24 hours later. Often engineers had to repair and use a captured enemy airfield. The German fields were well-built all-weather operations.[23]

Some of the Japanese island bases, built before the war, had excellent airfields. Most new Japanese installations in the Pacific were ramshackle affairs with poor siting, poor drainage, scant protection, and narrow, bumpy runways. Engineering was a low priority for the offense-minded Japanese, who chronically lacked adequate equipment and imagination. On a few islands, local commanders did improve aircraft shelters and general survivability, as they correctly perceived the danger of coming raids or invasions.[24] In the same theatre the United States Navy's own "construction battalions", collectively named the "Seabees" from the CB acronym adopted on the date of their formation in March 1942, would build over a hundred military airstrips and a significant degree of the military support infrastructure supplying the Pacific "island-hopping" campaign of the Allies during the Pacific war through 1945, as well as elsewhere in the world during the war years.

Tactical
Tactical air power involves gaining control of the airspace over the battlefield, directly supporting ground units (as by attacks on enemy tanks and artillery), and attacking enemy supply lines and airfields. Typically, fighter planes are used to gain air supremacy, and light bombers are used for support missions.[25]

Air supremacy

Gun camera photos of a Hawker Typhoon shooting down a Focke-Wulf Fw 190 in 1943
Tactical air doctrine stated that the primary mission was to turn tactical superiority into complete air supremacy—to totally defeat the enemy air force and obtain control of its air space. This could be done directly through dogfights and raids on airfields and radar stations or indirectly by destroying aircraft factories and fuel supplies. Anti-aircraft artillery (called "ack-ack" by the British, "flak" by the Germans, and "Archie" by the World War I USAAS) could also play a role, but it was downgraded by most airmen. The Allies won air supremacy in the Pacific in 1943 and in Europe in 1944.[26] That meant that Allied supplies and reinforcements would get through to the battlefront, but not the enemy's. It meant the Allies could concentrate their strike forces wherever they pleased and overwhelm the enemy with a preponderance of firepower. This was the basic Allied strategy, and it worked.


P-51 Mustangs of the 375th Fighter Squadron, Eighth Air Force mid-1944
One of the most effective demonstrations of air supremacy by the Western Allies over Europe occurred in early 1944, when Lieutenant General Jimmy Doolittle, who took command of the US 8th Air Force in January 1944, only a few months later "released" the building force of P-51 Mustangs from their intended mission to closely escort the 8th Air Force's heavy bombers, after getting help from British aviators in selecting the best available aircraft types for the task. The USAAF's Mustang squadrons were then tasked to fly well ahead of the bombers' combat box defensive formations by some 75–100 miles (120–160 km) to basically clear the skies, in the manner of a sizable "fighter sweep" air supremacy mission, of any defensive presence over the Third Reich of the Luftwaffe's Jagdgeschwader single-seat fighter wings. This important change of strategy also coincidentally doomed both the twin-engined Zerstörer heavy fighters and their replacement, heavily armed Focke-Wulf Fw 190A Sturmbock forces used as bomber destroyers, each in their turn. This change in American fighter tactics began to have its most immediate effect with the loss of more and more of the Luftwaffe's Jagdflieger fighter pilot personnel[27] and fewer bomber losses to the Luftwaffe as 1944 wore on.

Air superiority depended on having the fastest, most maneuverable fighters, in sufficient quantity, based on well-supplied airfields, within range. The RAF demonstrated the importance of speed and maneuverability in the Battle of Britain (1940), when its fast Spitfire and Hawker Hurricane fighters easily riddled the clumsy Stukas as they were pulling out of dives. The race to build the fastest fighter became one of the central themes of World War II.

Once total air supremacy in a theatre was gained, the second mission was interdiction of the flow of enemy supplies and reinforcements in a zone five to fifty miles behind the front. Whatever moved had to be exposed to air strikes, or else confined to moonless nights. (Radar was not good enough for nighttime tactical operations against ground targets.) A large fraction of tactical air power focused on this mission.

Close air support

A gun camera photo of a Wehrmacht convoy being strafed by a Balkan Air Force P-51 Mustang in 1945
The third and lowest priority (from the AAF viewpoint) mission was "close air support" or direct assistance to ground units on the battlefront, which consisted of bombing targets identified by ground forces, and strafing exposed infantry.[28] Airmen disliked the mission because it subordinated the air war to the ground war; furthermore, slit trenches, camouflage, and flak guns usually reduced the effectiveness of close air support. "Operation Cobra" in July, 1944, targeted a critical strip of 3,000 acres (1,214 ha) of German strength that held up the US breakthrough out of Normandy.[29] General Omar Bradley, his ground forces stymied, placed his bets on air power. 1,500 heavies, 380 medium bombers and 550 fighter bombers dropped 4,000 tons of high explosives. Bradley was horrified when 77 planes dropped their payloads short of the intended target:

"The ground belched, shook and spewed dirt to the sky. Scores of our troops were hit, their bodies flung from slit trenches. Doughboys were dazed and frightened ... A bomb landed squarely on McNair in a slit trench and threw his body sixty feet and mangled it beyond recognition except for the three stars on his collar."[30]
The Germans were stunned senseless, with tanks overturned, telephone wires severed, commanders missing, and a third of their combat troops killed or wounded. The defence line broke; J. Lawton Collins rushed his VII Corps forward; the Germans retreated in a rout; the Battle of France was won; air power seemed invincible. However, the sight of a senior colleague killed by error was unnerving, and after the completion of operation Cobra, Army generals were so reluctant to risk "friendly fire" casualties that they often passed over excellent attack opportunities that would be possible only with air support. Infantrymen, on the other hand, were ecstatic about the effectiveness of close air support:

"Air strikes on the way; we watch from a top window as P-47s dip in and out of clouds through suddenly erupting strings of Christmas-tree lights [flak], before one speck turns over and drops toward earth in the damnest sight of the Second World War, the dive-bomber attack, the speck snarling, screaming, dropping faster than a stone until it's clearly doomed to smash into the earth, then, past the limits of belief, an impossible flattening beyond houses and trees, an upward arch that makes the eyes hurt, and, as the speck hurtles away, WHOOM, the earth erupts five hundred feet up in swirling black smoke. More specks snarl, dive, scream, two squadrons, eight of them, leaving congealing, combining, whirling pillars of black smoke, lifting trees, houses, vehicles, and, we devoutly hope, bits of Germans. We yell and pound each other's backs. Gods from the clouds; this is how you do it! You don't attack painfully across frozen plains, you simply drop in on the enemy and blow them out of existence."[31]
Some forces, especially the United States Marine Corps, emphasized the air-ground team. The airmen, in this approach, also are infantrymen who understand the needs and perspective of the ground forces. There was much more joint air-ground training, and a given air unit might have a long-term relationship with a given ground unit, improving their mutual communications.[32]

In North-West Europe, the Allies used the "taxi-rank" (or "Cab-rank") system for supporting the ground assault. Fighter-bombers, such as the Hawker Typhoon or P-47 Thunderbolt, armed with cannon, bombs and rockets would be in the air at 10,000 ft over the battlefield. When support was required it could be quickly summoned by a ground observer. While often too inaccurate against armoured vehicles, rockets had a psychological effect on troops and were effective against the supply-carrying trucks used to support German tanks.
"""
```

**Comment by micoscriptor** [comment]
Should have mentioned also I am on ollama version 0.1.25
Running on Apple M1 8GB

**Comment by twwch** [comment]
I encountered the same problem, after fine-tuning the qwen-7b-chat model and using the method in this document to convert the model, the output was abnormal

https://github.com/ollama/ollama/blob/main/docs/import.md#importing-pytorch--safetensors

In addition, if the qwen fine-tuning model is used, there may be a situation where the output is not interrupted
